legislative/Q54830869/Q54844440/popolo-m17n.json

```diff
@@ -6,7 +6,102 @@
 
   ],
   "areas": [
-
+    {
+      "id": "Q2118",
+      "identifiers": [
+        {
+          "scheme": "MS_FB",
+          "identifier": "country:cl/region:an"
+        },
+        {
+          "scheme": "wikidata",
+          "identifier": "Q2118"
+        }
+      ],
+      "associated_wikidata_positions": [
+        "Q54830867"
+      ],
+      "type": {
+        "lang:es": "región de Chile",
+        "lang:en": "region of Chile"
+      },
+      "name": {
+        "lang:es": "REGIÓN DE ANTOFAGASTA"
+      },
+      "parent_id": null
```

Comment:
These are null because this branch isn't based off of country, which would provide the parent.